src/components/program/ProgramNotFound.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import { Helmet } from 'react-helmet';
+import { AppContext } from '@edx/frontend-platform/react';
+import { Container } from '@edx/paragon';
+
+const ProgramNotFoundPage = () => {
+  const { enterpriseConfig } = useContext(AppContext);
+
+  let PAGE_TITLE = 'Program not found';
+  if (enterpriseConfig) {
+    PAGE_TITLE += ` - ${enterpriseConfig.name}`;
+  }
+
+  return (
+    <Container size="lg" className="mt-3">
+      <Helmet title={PAGE_TITLE} />
+      <div className="text-center py-5">
+        <h1>Program not found</h1>
```

Comment:
nit: Since we already have this string saved in `PAGE_TITLE` above, maybe we could use it here too?

Reply:
that will be different from it as PAGE_TITLE will have enterprise info as well.